signup/views/users.py

```diff
@@ -77,11 +77,20 @@ def get_context_data(self, **kwargs):
         setattr(context['user'], 'full_name', context['user'].get_full_name())
         # URLs for user
         if is_authenticated(self.request):
+            try:
+                contact = Contact.objects.get(user=self.object)
+            except Contact.DoesNotExist:
+                # TODO not sure how to handle this
```

Comment:
See Django `get_or_create` method on models.

Reply:
OK, but is it ok that I'm automatically creating a `Contact` object each time it doesn't exist for a current user?

Reply:
It is OK, also something we do in other parts of the code base if I recall correctly.